Test.js

```diff
@@ -65,10 +76,40 @@ var loadProblemsFiles = (folder) => {
   });
 };
 
+const get_missing_tests = async function () {
+  const tests = await get_all_tests(TESTS_FOLDERS);
+  const problems = await get_all_tests(PROBLEMS_FOLDERS);
+
+  const hasTestStatus = problems.reduce((status, problemPath) => {
+    const baseIndex = PROBLEMS_FOLDERS.findIndex((basePath) =>
+      problemPath.startsWith(basePath)
+    );
+
+    let testPath = problemPath
+      .replace(PROBLEMS_FOLDERS[baseIndex], TESTS_FOLDERS[baseIndex])
+      .replace(/\.js$/, "_Test.js");
+    
+    status.push({
+      problem: problemPath,
+      hasTest: tests.includes(testPath)
+    });
+
+    return status;
+  }, []);
+  const missingTests = hasTestStatus.filter((stat) => !stat.hasTest);
+  console.log("Total Problems:", problems.length);
+  console.log("Missing Tests:", missingTests.length);
+
+  if(missingTests.length) {
+    console.table(missingTests);
+  }
+};
+
 if (process.argv.length > 2) {
   const path = process.argv.pop();
   solve(path);
 } else {
   test_all();
+  get_missing_tests();
```

Comment:
Both call `get_all_tests`, we could load them outside the class and pass them as arguments:

```js
  const tests = await get_all_tests(TESTS_FOLDERS);
  const problems = await get_all_tests(PROBLEMS_FOLDERS);
  
  run_all_tests(tests)
  get_missing_tests(tests, problems)
```

Reply:
good catch, updated ;)